01_fetch/src/fetch_usgs.py

```diff
@@ -0,0 +1,31 @@
+import os
+import urllib
+
+def fetch_params(outfile):
+    '''get table of all possible USGS site parameters'''
+    params_url = 'https://help.waterdata.usgs.gov/code/parameter_cd_query?fmt=rdb&group_cd=%'
+    urllib.request.urlretrieve(params_url, outfile)
+
+def fetch_data(site_list, start_dt, end_dt, outfile):
+    '''fetch USGS NWIS data for locations in site_list (gets all parameters available)'''
+    for site_num in site_list:
```

Comment:
Do you think putting the for loop in the function is better than looping over the function? I'm not sure, just thinking about it from a readability standpoint?

Reply:
For me, they are both equally readable. It is nice that the outfile gets formatted before putting it into the function if we pull the for loop out. I've pull it our here and in the munge function.